app/client/views/checkin/checkinCtrl.js

```diff
@@ -182,7 +182,7 @@ angular.module('reg')
       };
 
       function _apply(e){
-        AuthService.createAccount($scope.newUser.email, $scope.newUser.password, function success(user) {
+        AuthService.createAccount($scope.newUser.email, $scope.newUser.password, $scope.newUser.profile, function success(user) {
```

Comment:
Hmm why is there a createAccount call during the checkin portion?

Reply:
I think this might be for the admin to create users during check-in (for walk-in hackers)